enterprise/models.py

```diff
@@ -157,6 +158,22 @@ def get_default_customer_type():
     return enterprise_customer_type.id
 
 
+def get_default_site():
+    """
+    Get default site id to use when creating a new EnterpriseCustomer model.
+    The default value depending on what environment the person is in.
+    In production, it should be 'courses.edx.org'.
+    In stage it should be 'courses.stage.edx.org'.
+    """
+    ENVIRONMENT = os.getenv('DJANGO_ENVIRONMENT', 'development')
```

Comment:
rather than hard-coding the options here, we should add a configuration setting and/or reuse an existing one such as [LMS_BASE](https://github.com/openedx/edx-platform/blob/cb3a0e91e9b3e39ad5817c5e04abc8c1c9ce0f73/cms/envs/common.py#L770)

That way is more flexible - it can run in any environment (devstack, hosted devstack, sandboxes, etc) where the domains might vary widely, as well as any opensource community install.